tools/taskfile-swc.js

```diff
@@ -0,0 +1,70 @@
+// Based on Next.js swc taskr file.
+// https://github.com/vercel/next.js/blob/5378db8f807dbb9ff0993662f0a39d0f6cba2452/packages/next/taskfile-swc.js
+
+const path = require('path');
+
+const transform = require('@swc/core').transform;
+
+module.exports = function (task) {
+  task.plugin('swc', {}, function* (file, environment, { stripExtension } = {}) {
+    // Don't compile .d.ts
+    if (file.base.endsWith('.d.ts')) return;
+
+    const setting = {
+      output: 'build',
+      options: {
+        module: {
+          type: 'commonjs',
+        },
+        env: {
+          targets: {
+            node: '12.13.0',
```

Comment:
```suggestion
            node: '16.9.1',
```

I think that we can target Node 16 here safely, if not 18.

Reply:
A quick pass showed that we use 14 in some actions.

Reply:
Yeah, there is a bit of debt in the workspace, but for tools we use Node 16 typings and the tools action runner have Node 16 bundled - https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md